staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -92,14 +92,14 @@ type KubectlOptions struct {
 	genericiooptions.IOStreams
 }
 
-var defaultConfigFlags = genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag().WithDiscoveryBurst(300).WithDiscoveryQPS(50.0)
+var DefaultConfigFlags = genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag().WithDiscoveryBurst(300).WithDiscoveryQPS(50.0)
 
 // NewDefaultKubectlCommand creates the `kubectl` command with default arguments
 func NewDefaultKubectlCommand() *cobra.Command {
 	return NewDefaultKubectlCommandWithArgs(KubectlOptions{
 		PluginHandler: NewDefaultPluginHandler(plugin.ValidPluginFilenamePrefixes),
 		Arguments:     os.Args,
-		ConfigFlags:   defaultConfigFlags,
+		ConfigFlags:   DefaultConfigFlags,
```

Comment:
When we expose this, modifying values and maintaining it might be tricky and currently it is safe when it is unexported. I think, you can copy it as how we did it in [oc](https://github.com/openshift/oc/blob/705ef3fd163df4c2763fce653151b68e2ad3a8f0/pkg/cli/cli.go#L99). 
/hold

Reply:
Not sure I understand what is tricky/unsafe here. I am guessing you mean exporting the variable would mean it might conflict with some other variable with a similar name in the future? 

Doing something like https://github.com/openshift/oc/blob/705ef3fd163df4c2763fce653151b68e2ad3a8f0/pkg/cli/cli.go#L99-L101 would mean we'd have to update the hardcoded value every time there is a change in the upstream burst/qps. 

Instead if we can write 
```go
func defaultConfigFlags() *genericclioptions.ConfigFlags {
	return genericclioptions.DefaultConfigFlags
}
```

We won't have to change the code above every time there is a bump in the discovery QPS/Burst (or any other change in the default config). 

It doesn't have to be an exported variable. I am open to any other option as well which would allow using the default config flags in projects like oc, argo workflows and others without hardcoding them.